app/assets/javascripts/carto_reference.js

```diff
@@ -0,0 +1,3 @@
+$(document).ready(function() {
```

Comment:
If we don't remove it then it stays in the DOM empty which I don't think is correct. I've noticed for restricted maps because they don't have a carto_reference there is also an empty li tag with class exports. 
If you check https://github.com/geoblacklight/geoblacklight/blob/3603575c85b81b4070d56f3011dd9b55873a2697/spec/features/exports_spec.rb
exports is only used for the Open Carto.

Reply:
Also I've noticed for maps where there is no 'Download' button the DOM has an empty li tag with class 'downloads'. My opinion is that if we don't use them we should not create the element.

Reply:
I'm concerned about upstream changes to GeoBlacklight that this change might block. If for example, a new export was added, it might be difficult to discover why it doesn't appear. I'm also concerned about more javascript functions in an app that is already javascript heavy. I'm not totally set on this, but I feel like there should be a good reason to do it. Are there accessibility concerns or performance concerns? My gut feeling is that an empty tag is more preferable in terms of performance than another piece of javascript that is run on every page load. @axamei @tampakis @jrgriffiniii Do you have any thoughts about this?

Reply:
From this point of view, that GeoBlacklight might introduce more 'exports' or 'downloads' and we want an li tag to be ready to welcome new features then I would agree that it should stay. I agree that the specific JS will remove the li after it's been created and maybe it would be better not to create them at all in the first place, if they don't exist as attributes in the solr document.

Reply:
@christinach Maybe better approach is just to comment this line out and a note as to why.

https://github.com/pulibrary/pulmap/blob/master/app/controllers/catalog_controller.rb#L252

Reply:
I think @eliotjordan's approach is a good idea.

Reply:
Perfect @eliotjordan this is what I was looking from the beginning on this issue.

Reply:
In this case we don't even need to nil the carto_reference attribute.